src/types/Config.tsx

```diff
@@ -858,6 +859,7 @@ export const userMenuCompany = [
   PAGES.COMPANY_CERTIFICATE,
   PAGES.COMPANY_WALLET,
   PAGES.COMPANY_DATA,
+  PAGES.MANAGEMENT_ONBOARDING_SERVICE_PROVIDER,
```

Comment:
naming appears to inconsistent:

```suggestion
  PAGES.ONBOARDING_SERVICE_PROVIDER_MANAGEMENT,
```